Cargo.lock

```diff
@@ -631,9 +631,9 @@ dependencies = [
 
 [[package]]
 name = "hashbrown"
-version = "0.14.5"
```

Comment:
This contains breaking changes https://github.com/rust-lang/hashbrown/blob/master/CHANGELOG.md#v0150---2024-10-01

> This update contains breaking changes that remove the raw API with the hope of centralising on the HashTable API in the future. You can follow the discussion and progress in #545 to discuss features you think should be added to this API that were previously only possible on the raw API.